jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/OpenFalconWriter.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * The MIT License
+ * Copyright (c) 2010 JmxTrans team
+ * <p/>
```

Comment:
the `<p/>` does not match the header template. You can use `mvn license:format` to automatically format the files.